x/reward/types/action_builder.go

```diff
@@ -228,6 +230,26 @@ func (v *VoteActionBuilder) GetEventCriteria() *EventCriteria {
 	})
 }
 
+// govVoteMsgURLs the MsgURLs of all the governance module's vote messages.
+// Use getGovVoteMsgURLs() instead of using this variable directly.
+var govVoteMsgURLs []string
+
+// getGovVoteMsgURLs returns govVoteMsgURLs, but first sets it if it hasn't yet been set.
+func getGovVoteMsgURLs() []string {
+	if len(govVoteMsgURLs) != 4 {
```

Comment:
Same comment as what I left before. Is it possible to switch the condition to govVoteMsgURLs == nil?

Reply:
Yup. Just changed it.